File: client/lib/common/models/invitation.dart
class Invitation {
  int orgId;
  String email;
  int roleId;

  Invitation({required this.orgId, required this.email, required this.roleId});

  Invitation.fromJson(dynamic data)
    : orgId = data['orgId'],
      email = data['email'],
      roleId = data['roleId'];
}

File: server/flaskr/models/data/invitation.py
from dataclasses import dataclass


@dataclass(slots=True)
class Invitation:
    orgId: int
    email: str
    roleId: int

    def to_dict(self) -> dict[str, str | int]:
        return {"orgId": self.orgId, "email": self.email, "roleId": self.roleId}
